mne/event.py

```diff
@@ -885,3 +886,402 @@ def concatenate_events(events, first_samps, last_samps):
+
+    # new versions only (DACQ >= 3.4)
+    _dacq_vars = _dacq_vars_compat + ('magMax', 'magMin', 'magNoise',
+                                      'magSlope', 'magSpike', 'version')
+
+    _event_vars = _event_vars_compat + ('Name', 'Channel', 'NewBits',
+                                        'OldBits', 'NewMask', 'OldMask')
+
+    def __init__(self, info):
+        acq_pars = info['acq_pars']
+        if not acq_pars:
+            raise ValueError('No acquisition parameters')
+        self.acq_dict = self._acqpars_dict(acq_pars)
+        if 'ERFversion' in self.acq_dict:
+            self.compat = False  # DACQ ver >= 3.4
+        elif 'ERFncateg' in self.acq_dict:  # probably DACQ < 3.4
+                self.compat = True
+        else:
+            raise ValueError('Cannot parse acquisition parameters')
+        dacq_vars = self._dacq_vars_compat if self.compat else self._dacq_vars
+        # set instance variables
+        for var in dacq_vars:
+            val = self.acq_dict['ERF' + var]
+            if var[:3] in ['mag', 'meg', 'eeg', 'eog', 'ecg']:
+                val = float(val)
+            elif var in ['ncateg', 'nevent']:
+                val = int(val)
+            setattr(self, var.lower(), val)
+        self.stimsource = (
+            'Internal' if self.stimsource == '1' else 'External')
+        # collect all events and categories
+        self._events = self._events_from_acq_pars()
+        self._categories = self._categories_from_acq_pars()
+        # mark events that are used by a category
+        for cat in self._categories.values():
+            if cat['event']:
+                self._events[cat['event']]['in_use'] = True
+            if cat['reqevent']:
+                self._events[cat['reqevent']]['in_use'] = True
+        # make mne rejection dicts based on the averager parameters
+        self.reject = {'grad': self.megmax, 'eeg': self.eegmax,
+                       'eog': self.eogmax, 'ecg': self.ecgmax}
+        if not self.compat:
+            self.reject['mag'] = self.magmax
+        self.reject = {k: float(v) for k, v in self.reject.items()
+                       if float(v) > 0}
+        self.flat = {'grad': self.megmin, 'eeg': self.eegmin}
+        if not self.compat:
+            self.flat['mag'] = self.magmin
+        self.flat = {k: float(v) for k, v in self.flat.items()
+                     if float(v) > 0}
+
+    def __repr__(self):
+        s = '<AcqParserFIF | '
+        s += 'categories: %d ' % self.ncateg
+        cats_in_use = len(self._categories_in_use)
+        s += '(%d in use), ' % cats_in_use
+        s += 'events: %d ' % self.nevent
+        evs_in_use = len(self._events_in_use)
+        s += '(%d in use)' % evs_in_use
+        if self.categories:
+            s += '\nAveraging categories:'
+            for cat in self.categories:
+                s += '\n%d: "%s"' % (cat['index'], cat['comment'])
+        s += '>'
+        return s
+
+    def __getitem__(self, items):
```

Comment:
Please document this method 

https://2598-1301584-gh.circle-artifacts.com/0//home/ubuntu/mne-python/doc/_build/html/generated/mne.AcqParserFIF.html#mne.AcqParserFIF

Take a look how we do it with epochs